ansible/roles/proteus-events/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: create Proteus group
+  group: name={{ proteus_group }}  state=present
+
+- name: create Proteus user
+  user:
+    name: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    createhome: no
+    shell: /sbin/nologin
+    comment: "Proteus User"
+    state: present
```

Comment:
`/sbin/nologin` is not a valid shell.

UID will clash with superq as soon as dom0 with node_exporter is deployed there:
```
events.proteus.test.ooni.io | SUCCESS | rc=0 | (stdout) uid=10007(proteus) gid=10007(proteus) groups=10007(proteus)
notify.proteus.test.ooni.io | SUCCESS | rc=0 | (stdout) uid=10007(proteus) gid=10007(proteus) groups=10007(proteus)
proteus.test.ooni.io | SUCCESS | rc=0 | (stdout) uid=10007(proteus) gid=10007(proteus) groups=10007(proteus)
registry.proteus.test.ooni.io | SUCCESS | rc=0 | (stdout) uid=10007(proteus) gid=10007(proteus) groups=10007(proteus)
notify.proteus.ooni.io | SUCCESS | rc=0 | (stdout) uid=10006(proteus) gid=10006(proteus) groups=10006(proteus)
registry.proteus.ooni.io | SUCCESS | rc=0 | (stdout) uid=10006(proteus) gid=10006(proteus) groups=10006(proteus)
events.proteus.ooni.io | SUCCESS | rc=0 | (stdout) uid=10006(proteus) gid=10006(proteus) groups=10006(proteus)
proteus.ooni.io | SUCCESS | rc=0 | (stdout) uid=10006(proteus) gid=10006(proteus) groups=10006(proteus)
```